EU4toV2/Source/V2World/Event/Event.h

```diff
@@ -0,0 +1,53 @@
+#ifndef EVENT_H
+#define EVENT_H
+
+#include <random>
+#include <string>
+#include <unordered_set>
+#include <vector>
+#include "../../V2World/Province/Province.h"
+#include "Parser.h"
+
+namespace V2
+{
+
+        class Event : commonItems::parser
+        {
+        public:
+                Event(std::istream& theStream);
+                void
+                shuffle(const std::map<int, std::shared_ptr<Province>>& provMap,
+                        std::default_random_engine& shuffler);
+                friend std::ostream& operator<<(std::ostream& output,
+                                                const Event& event);
+
+        private:
+                // Vicky event ID.
```

Comment:
In general, no need for comments describing the variables. If its unclear, aim for a more descriptive name. If its still unclear, go ahead with a comment.